ci_framework/roles/edpm_build_images/defaults/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_edpm_build_images"
+
+cifmw_edpm_build_images_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework') }}"
+cifmw_edpm_build_images_src: "https://github.com/openstack-k8s-operators/edpm-image-builder"
+cifmw_edpm_build_images_artifacts: "{{ cifmw_edpm_build_images_basedir }}/artifacts"
+cifmw_edpm_build_images_packages:
```

Comment:
are you installing these deps so that you can build the ioroic ipa image with dib? in which case these are the build host packaged no the image packages?

Reply:
@SeanMooney Hi, Thanks for the review.

No no. I am installing these packages on zuul host and not in the image or edpm host.

Reply:
@bshewale We should name this variable correctly to avoid confusion.